### PR TITLE
Add document of emoji

### PR DESCRIPTION
**Pull request by moko256**

### CODE REVIEW

File: Using-the-API/API.md [hunk 1]
```diff
@@ -640,6 +641,14 @@ ___
 | `ancestors`              | The ancestors of the status in the conversation, as a list of [Statuses](#status)   | no       |
 | `descendants`            | The descendants of the status in the conversation, as a list of [Statuses](#status) | no       |
 
+### Emoji
+
+| Attribute                | Description                        | Nullable |
+|--------------------------|------------------------------------|----------|
+| `shortcode`              | The shortcode of the emoji         | no       |
+| `static_url`             | URL to the emoji static image      | no       |
+| `url`                    | URL to the emoji image             | no       |
```
Comment (nightpool) [inline Using-the-API/API.md]: You might want to be a bit more clear about the difference between static_url and url here. maybe something like "URL to the emoji image (may be animated)" for `url` and "URL to the static emoji image, with animation stripped (auto-generated)"?

File: Using-the-API/API.md [hunk 2]
```diff
@@ -716,6 +725,7 @@ ___
 | `reblog`                 | `null` or the reblogged [Status](#status)                                     | yes      |
 | `content`                | Body of the status; this will contain HTML (remote HTML already sanitized)    | no       |
```
Comment (nightpool) [inline Using-the-API/API.md]: You might want to mention here that it may now contain shortcodes, which should be replaced with the images in "emojis"

Comment (moko256) [reply Using-the-API/API.md]: Ok. I see. Thank you

**Comment by nightpool** [comment]
thanks for the contribution! there are a few more things to clean up, but I'm gonna merge this as is so we have *something* there and follow it up in a further PR

**Pull request merged**